<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>raporty</class>
 <widget class="QDialog" name="raporty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zestawienie miesięczne</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Zakres dat</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_data_od">
           <property name="text">
            <string>data od</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_od"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_data_od">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_data_do">
           <property name="text">
            <string>do</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_do"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_data_do">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_suma_punktow">
           <property name="text">
            <string>suma punktów</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_suma_punktow">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTableView" name="tableView_osrodki_punkty">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_wybor"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_wybor">
        <property name="text">
         <string>Wybrany ośrodek</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_druk">
           <property name="text">
            <string>Raport miesieczy ogólny</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_druk2">
           <property name="text">
            <string>Raport miesięczny z wybranego ośrodka</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
